src/persist/src/postgres.rs

```diff
@@ -127,21 +131,21 @@ impl PostgresConsensus {
         // TODO: reconsider opening with NoTLS. Perhaps we actually want to,
         // especially given the fact that its not entirely known what data
         // will actually be stored in Consensus.
-        let (mut client, conn) = tokio_postgres::connect(&config.url, NoTls).await?;
-        let handle = task::spawn(|| "pg_consensus_client", async move {
-            if let Err(e) = conn.await {
-                tracing::error!("connection error: {}", e);
-                return;
-            }
-        });
-
+        let mgr = Manager::new(config.url.parse()?, NoTls);
+        let pool = Pool::builder(mgr)
+            .max_size(config.pool_max_size)
```

Comment:
Having a `min_size` would be cool (sadly doesn't seem to exist) because we could get more of a handle of connection limit issues instead of surprisingly hitting them when too many pools all of a sudden choose to make a bunch of connections.

Reply:
yeah, I think that's interesting. one way maybe to hack it is to call `pool.get().await` `min-size` times in the constructor (without letting any of them drop until you've gotten all of them)